src/store/reducers/list.js

```diff
@@ -13,6 +13,9 @@ const list = (state = initialState, action) => {
                 ...state,
                 tasks: state.tasks.filter((task, id) => id !== action.payload.id ),
             }
+        case SAVE:
+            localStorage.setItem('list',JSON.stringify(state))
```

Comment:
it should be in actions file, inside of the the function, which returns the action generator function